packages/camera/example/ios/Flutter/flutter_assets/LICENSE

```diff
@@ -0,0 +1,17825 @@
+analyzer
```

Comment:
Don't check in the `flutter_assets` folder. Add to .gitignore.

Reply:
done